src/components/form/Checkbox/CheckboxField.tsx

```diff
@@ -1,4 +1,5 @@
 import { FormikProps } from 'formik'
+import _get from 'lodash/get'
```

Comment:
Are the changes in this file related to this PR ?

Reply:
No, but I saw it so I changed it